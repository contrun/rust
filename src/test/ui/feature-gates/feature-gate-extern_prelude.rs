can-only-test-this-in-run-make-fulldeps //~ ERROR expected item, found `can`
