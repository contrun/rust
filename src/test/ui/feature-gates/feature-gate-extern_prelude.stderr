error: expected item, found `can`
  --> $DIR/feature-gate-extern_prelude.rs:1:1
   |
LL | can-only-test-this-in-run-make-fulldeps
   | ^^^ expected item

error: aborting due to previous error

